Makefile

```diff
@@ -112,3 +112,10 @@ prepare_bots_prod:
 
 run_bots: prepare_bots
 	poetry run python main.py --config "$(CONFIG_FILE)"
+
+run_bots_memray: prepare_bots
+	poetry run \
+		python \
+		-m memray run \
+		$(MEMRAY_ARGS) \
```

Comment:
Let's define default value for `MEMRAY_ARGS` like here for `CONFIG_FILE`: https://github.com/vegaprotocol/research-bots/pull/6/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L5

Reply:
done